code/game/objects/items/storage/backpack.dm

```diff
@@ -889,6 +894,23 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 	fuel_type = "utnapthal"
 	has_gamemode_skin = TRUE
 
+/obj/item/storage/backpack/marine/engineerpack/flamethrower/afterattack(obj/target, mob/user, proximity)
```

Comment:
Are you sure you want to add this function? This currently allows a normally napthal fueled item with any fuel with no way of emptying it; and other fuels when used will cause either a to_chat message of `You can't mix fuel mixtures!` or `This chemical is not potent enough to be used in a flamethrower!`.

See `/obj/item/storage/backpack/marine/engineerpack/flamethrower/attackby` not only is that checking to ensure its naptha; but should just handle this regardless no?

Reply:
i should have commented this....
the idea was to overide above code on the specific child object that is being used for flamethrower backpack 
so that people can still use mix on them...
added comment